python/feathub/feature_views/feature_view.py

```diff
@@ -44,6 +46,10 @@ def __init__(
                          which refers to a feature in the source table.
         :param keep_source_fields: True iff all fields in the source table should be
                                    included in this table.
+        :param timestamp_field: Optional. If it is not None, it is the name of the field
+                                whose values show the time when the corresponding row
+                                is generated.
+        :param timestamp_format: The format of the timestamp field.
```

Comment:
This doc seems confusing. If a FeatureView is instantiated with `timestamp_field=None` and `timestamp_format=""%Y-%m-%d %H:%M:%S.%f""`, what is the expected format of the table descriptor represented by this FeatureView?